instrumentation/okhttp/okhttp-3.0/library/src/main/java/io/opentelemetry/instrumentation/okhttp/v3_0/OkHttpAttributesExtractor.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.okhttp.v3_0;
+
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpAttributesExtractor;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+final class OkHttpAttributesExtractor extends HttpAttributesExtractor<Request, Response> {
+  @Override
+  protected @Nullable String method(Request request) {
```

Comment:
Think some of these aren't `@Nullable` (sure about `request.url.toString()` at least :)

Reply:
Ouch, auto-generated method stubs... removed, thanks!